http.go

```diff
@@ -17,23 +17,37 @@ import (
 type httpServer struct {
 	ctx *Context
 
-	Requests     uint64
-	GetRequests  uint64
-	GetHits      uint64
-	GetMisses    uint64
+	Requests uint64
```

Comment:
Is this `go fmt` related? I think the go formatter generally tab-aligns everything inside of a struct, but perhaps if you include newlines between blocks it reset the tab alignment.